### ISSUE TITLE
Unexpected Keyword Argument

### ISSUE THREAD

**Issue by aguthrie19** [AnesBenmerzoug/depender#1]
Runtime error given a path as input:
```
$ pip3 install depender
$ depender /my/directory/path
Traceback (most recent call last):
  File "/usr/local/bin/depender", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
TypeError: main() got an unexpected keyword argument 'dims```

**Comment by AnesBenmerzoug** [comment]
@aguthrie19  Sorry for the late reply.
First of all, thanks for using this small package and reporting an issue.
It seems something changed in version 7.0 of click and I hadn't noticed it.
I changed the requirements to fix that and uploaded a new version, 0.1.2, of this package.

**Comment by aguthrie19** [comment]
Not a problem! Thank you for the fix.
